src/FsCheck.NUnit/AssemblyInfo.fs

```diff
@@ -1,13 +1,16 @@
-﻿// Auto-Generated by FAKE; do not edit
+
+// Auto-generated; edits may be deleted at any time
 namespace System
 open System.Reflection
+open System.Runtime.CompilerServices
+
+[<assembly: AssemblyTitle("FsCheck.NUnit")>]
```

Comment:
Regenerated these files from build.fsx; took the opportunity to delete the unnecessary word `Attribute`.